bot/exts/fun/madlibs.py

```diff
@@ -0,0 +1,116 @@
+import json
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+from typing import TypedDict
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
```

Comment:
See the comment below on line 86. If you want to accept that suggestion, you'd also want to accept this suggestion for it to work, as it requires an import from bot.constants.
```suggestion
from bot.constants import Colours, NEGATIVE_REPLIES
```